accredible_certificate/queue.py

```diff
@@ -239,11 +239,11 @@ def add_cert(
                         cert.status = defined_status
                         cert.key = json_response["credential"]["id"]
                         if 'private' in json_response:
-                            cert.download_url = "https://wwww.accredible.com/" + \
+                            cert.download_url = "https://wwww.credential.net/" + \
```

Comment:
Oh, I see now there were 4 `w`s in that url to begin with!  How was that ever working?
`"https://www.credential.net/"`

Reply:
Looks like the non-private URLs have been `wwww` ever since this code made 3 years ago

Reply:
Totally bonkers!